src/chrome/content/rules/Nintendolife.com.xml

```diff
@@ -0,0 +1,20 @@
+<ruleset name="Nintendolife.com">
+
+    <target host="nintendolife.com" />
+    <target host="www.nintendolife.com" />
+    <target host="3ds.nintendolife.com" />
+    <target host="3dsvc.nintendolife.com" />
+    <target host="3dsware.nintendolife.com" />
+    <target host="dsiware.nintendolife.com" />
+    <target host="ds.nintendolife.com" />
+    <target host="images.nintendolife.com" />
+    <target host="retro.nintendolife.com" />
+    <target host="static.nintendolife.com" />
+    <target host="vc.nintendolife.com" />
+    <target host="wii.nintendolife.com" />
+    <target host="wiiware.nintendolife.com" />
```

Comment:
```
fs.nintendolife.com
sp.nintendolife.com
```

Reply:
Those two url's do not resolve for me? That is why I removed them. Do they work for you?

Reply:
They WFM and some resources are loaded from there.